committee-steering/governance/working-group-formation-process.md

```diff
@@ -0,0 +1,87 @@
+# Kubernetes Working Group Formation Process - Proposal
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
+
+## Process Overview and Motivations
+As opposed to “Birds of a Feather” or discussion groups, Working Groups provide a more formal avenue for disparate
+groups to intellectually collaborate around a common problem, craft a balanced position, and disband. Because they
+represent the interests of multiple special interest groups, they are a vehicle for consensus building, not
+construction. By design, the code under discussion resides in either multiple parts of the project, or is in the
```

Comment:
I have no idea what "not construction" means. Does that mean that working groups don't write code? That doesn't seem correct...  The following sentence clarifies this somewhat, I think what you really mean is that the code that a working group may write will land in a variety of SIGs and not in a specific repo owned by the working group.

Reply:
Delete this